packages/cli/src/commands/styles/utils/figma.utils.ts

```diff
@@ -186,17 +187,16 @@ type EffectType = {
   showShadowBehindNode: boolean,
 };
 
-export type NodeDoc = EffectsNode & RectangleNode & NodeDocument & {
-
-};
+export type NodeDoc = EffectsNode & RectangleNode & NodeDocument & {};
 
 export type DesignToken = {
   type: 'typography' | 'color' | 'spacing' | 'breakpoint' | 'grid' | 'elevation',
   name: string,
   viewPort?: string,
   cascade?: boolean,
-  token: string,
+  token?: string,
```

Comment:
not too sure why `token` and `node_id` are optional. These are the only two values that need to be there to write the tokens to our css file